src/shared/lib/resources/cwu-opportunity.ts

```diff
@@ -0,0 +1,41 @@
+import { Addendum } from 'shared/lib/resources/addendum';
+import { FileRecord } from 'shared/lib/resources/file';
+import { User } from 'shared/lib/resources/user';
+import { Id } from 'shared/lib/types';
+
+export enum OpportunityStatus {
+  Draft = 'DRAFT',
+  Published = 'PUBLISHED',
+  Evaluation = 'EVALUATION',
+  Awarded = 'AWARDED',
+  Suspended = 'SUSPENDED',
+  Canceled = 'CANCELED'
+}
+
+export interface CWUOpportunity {
+  id: Id;
+  createdAt: Date;
+  createdBy: User;
+  updatedAt: Date;
+  updatedBy: User;
+  title: string;
+  teaser: string;
+  remoteOk: boolean;
+  remoteDesc: string;
+  location: string;
+  reward: number;
+  skills: string[];
+  description: string;
+  proposalDeadline: Date;
+  assignmentDate: Date;
+  startDate: Date;
+  completionDate: Date;
+  submissionInfo: string;
+  acceptanceCriteria: string;
+  evaluationCriteria: string;
+  status: OpportunityStatus;
+  attachments: FileRecord[];
+  addenda: Addendum[];
+}
+
+export type CWUOpportunitySlim = Omit<CWUOpportunity, 'attachments' | 'addenda' | 'updatedAt' | 'updatedBy'>;
```

Comment:
For now, this can simply be:

```ts
export type CWUOpportunitySlim = Pick<CWUOpportunity, 'title' | 'createdAt' | 'createdBy' | 'updatedAt' | 'updatedBy' | 'status' | 'proposalDeadline'>;
```

Reply:
We may need to modify this in late Feb/March when we get the design for the opportunity list page.